src/main/java/uk/gov/hmcts/sptribs/document/DocumentUtil.java

```diff
@@ -92,17 +94,17 @@ public static void uploadDocument(CaseData data) {
     public static List<String> validateUploadedDocuments(List<ListValue<CaseworkerCICDocument>> uploadedDocuments) {
         List<String> errors = new ArrayList<>();
 
-        if (uploadedDocuments != null && !uploadedDocuments.isEmpty()) {
+        if (!CollectionUtils.isEmpty(uploadedDocuments)) {
```

Comment:
Can we use CollectionUtils.isNotEmpty instead for better readability and consistency

Reply:
Don't think isNotEmpty is available. Are you referring to use some other package?

Reply:
Thanks - updated.